sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using TensorFlow in SageMaker - Quickstart\n",
```

Comment:
I don't think it's clear why this is called "Quickstart" - are other TF notebooks not quick ways to start using TF on SageMaker?

Reply:
eh, I guess that's fine